README.md

```diff
@@ -12,60 +12,58 @@ platform.
 
```

Comment:
"is a free and open source software which provides a" -> "is a free and open source source software **system** which provides a"


Also, elaborate on what "unified push service" means. It's more like "push service with an API wrapping the APIs of various push notification services"

One way to rephrase: add the below sentence:

- The `uniqush-push` API abstracts the APIs of the various push services used to send notifications to those devices."